df.units.xml

```diff
@@ -2324,22 +2324,22 @@
         <int32_t name='unk_2c'/>
         <int32_t name='attack_velocity'/>
         <bitfield name='flags' base-type='int32_t'>
-            <flag-bit/>
-            <flag-bit/>
-            <flag-bit/>
-            <flag-bit/>
-            <flag-bit/>
-            <flag-bit name='quick'/>
-            <flag-bit name='heavy'/>
-            <flag-bit name='wild'/>
-            <flag-bit name='precise'/>
-            <flag-bit name='charge'/>
-            <flag-bit comment='multi-attack'/>
-            <flag-bit/>
-            <flag-bit name='lightly_tap'/>
-            <flag-bit/>
-            <flag-bit/>
-            <flag-bit name='spar_report'/>
+            <flag-bit name='force_small_aim_minus' comment='(FORCE_SMALL_AIM_MINUS)'/>
+            <flag-bit name='can_get_momentum_bonus' comment='(CAN_GET_MOMENTUM_BONUS)'/>
+            <flag-bit name='no_parry' comment='(NO_PARRY)'/>
+            <flag-bit name='no_block' comment='(NO_BLOCK)'/>
+            <flag-bit name='does_not_stop_wait' comment='(DOES_NOT_STOP_WAIT)'/>
+            <flag-bit name='quick' comment='(QUICK)'/>
+            <flag-bit name='heavy' comment='(HEAVY)'/>
+            <flag-bit name='wild' comment='(WILD)'/>
+            <flag-bit name='precise' comment='(PRECISE)'/>
+            <flag-bit name='charge' comment='(INDEPENDENT_MULTIATTACK)'/>
+            <flag-bit name='bad_multiattack' comment='(BAD_MULTIATTACK)'/>
+            <flag-bit name='zero_criticals' comment='(ZERO_CRITICALS)'/>
+            <flag-bit name='lightly_tap' comment='(NO_LETHAL_INTENT)'/>
+            <flag-bit name='automatic_hit' comment='(AUTOMATIC_HIT)'/>
+            <flag-bit name='sparring_hit' comment='(SPARRING_HIT)'/>
+            <flag-bit name='spar_report' comment='()'/>
```

Comment:
Should `charge` be renamed `independent_multiattack`?
What's the Bay12 name for `spar_report`?

Reply:
I'd say keep `charge` as it is.

There does not appear to be a bit for `spar_report` -- the header just goes up to SPARRING_HIT. That flag bit may need to be removed

Reply:
i can't find any indication for a use of bit15 and believe we can safely remove `spar_report`

Reply:
> I'd say keep `charge` as it is.

Does it correctly describe the behavior? I.e., is the bit set during a charge?

Reply:
We just have a general policy of "if it's named, then don't change it unless it's egregious". Behavior notes can go in the comment.